manifests/s/Schuischta/SystemLoadTracker/1.1/Schuischta.SystemLoadTracker.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Schuischta.SystemLoadTracker
+PackageVersion: "1.1"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: SystemLoadTracker.exe
+  PortableCommandAlias: portable
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

(Automated response - build 808.)

---

Hi @stephengillie,

do i have to change something? If so what exactly do i have to change?

Thanks!